sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -657,7 +657,7 @@ object GpuCast extends Arm {
    *
    * when `legacyCastToString = false`, step 2, 5 are skipped
    */
-  private  def castArrayToString(input:                    ColumnView,
+  private def castArrayToString(input:                    ColumnView,
                                  elementType:                DataType,
                                  ansiMode:                    Boolean,
                                  legacyCastToString:          Boolean,
```

Comment:
These lines are now mis-indented due to the whitespace fix above.

Reply:
Thank you. Has been fixed.